infra/util/pom.xml

```diff
@@ -60,16 +60,16 @@
                         <goals>
                             <goal>copy</goal>
                         </goals>
-                        <phase>package</phase>
+                        <phase>process-test-classes</phase>
                         <configuration>
                             <artifactItems>
                                 <artifactItem>
                                     <groupId>org.apache.shardingsphere</groupId>
-                                    <artifactId>shardingsphere-infra-util-groovy</artifactId>
-                                    <version>${project.version}</version>
+                                    <artifactId>shardingsphere-infra-util</artifactId>
+                                    <version>5.3.1</version>
                                     <type>jar</type>
                                     <overWrite>true</overWrite>
-                                    <destFileName>shardingsphere-infra-util-groovy.jar</destFileName>
+                                    <destFileName>shardingsphere-infra-util.jar</destFileName>
                                 </artifactItem>
```

Comment:
- This is another processing idea that I pointed out at https://github.com/apache/shardingsphere/pull/23873#discussion_r1091862118 .